### ISSUE TITLE
How i load a local game

### ISSUE THREAD

**Issue by vanszeitboy** [endrift/gbajs#31]
Hi. How i change these parts of code for loading a local game:

<input type="file" onchange="run(this.files[0]);">

function run(file) {
 gba.loadRomFromFile(file, function(result) {
  if (result) {
   for (var i = 0; i < runCommands.length; ++i) {
    runCommands[i]();
  }
  runCommands = [];
  gba.runStable();
  }
 });
}

**Comment by DominicBridge** [comment]
Did you ever make any progress with this? I've tried a few things but nothing seems to be working.

**Comment by endrift** [comment]
GBA.js is unmaintained. Also, by local, do you mean one off of the file system of the computer without you using a file select dialog? Because that's impossible.

**Comment by DominicBridge** [comment]
Completely? Yeah I was looking for a way to get this running on the Xbox One Browser, which also supports gamepad so was wondering if I had a rom hosted on the same server as a the emulator whether that could work.

Thanks for the quick reply!

**Comment by endrift** [comment]
If it's off the same *remote* server, e.g. they're both on an http site, then it can be done, but I've intentionally not added support for that to prevent this being used as a drop in thing for online ROM sites.

If one or both are on a file:/// page, it's impossible.

**Comment by DominicBridge** [comment]
Yeah the plan is on the remote server as the Xbox One lacks as ability to upload. 

But yeah, I completely understand the reasoning. I'll take a harder look in to it.

Cheers!

**Comment by overlox** [comment]
index.html
```

		window.onload = function() {
... 
				loadGame('resources/rom.gba', function(rom) {
					run(rom);
				});
```



resources/xhr.js
```
function loadGame(url, callback) {
	var xhr = new XMLHttpRequest();
	xhr.open('GET', url);
	xhr.responseType = 'blob';

	xhr.onload = function() { callback(xhr.response) };
	xhr.send();
}
```
